### PR TITLE
Add Export Macro for DLL Generation

### PR DESCRIPTION
**Pull request by ptsouchlos**
See #1

### CODE REVIEW

**Comment by teekae** [comment]
Thanks for this PR, will try and test it soon

File: blinkstickcpp/CMakeLists.txt [hunk 1]
```diff
@@ -4,9 +4,13 @@ add_library(blinkstickcpp
     src/device.cpp
 )
 
+include(GenerateExportHeader)
+generate_export_header(blinkstickcpp)
```
Comment (teekae) [inline blinkstickcpp/CMakeLists.txt]:
```suggestion
generate_export_header(blinkstickcpp EXPORT_FILE_NAME blinkstick/export.hpp)
```

File: blinkstickcpp/CMakeLists.txt [hunk 2]
```diff
@@ -24,7 +28,12 @@ install(TARGETS blinkstickcpp EXPORT blinkstickcppTargets
         ARCHIVE DESTINATION "${INSTALL_LIB_DIR}"
         LIBRARY DESTINATION "${INSTALL_LIB_DIR}")
 
-install(FILES blinkstick.hpp device.hpp DESTINATION "${INSTALL_INC_DIR}")
+install(FILES 
+            blinkstick.hpp
+            device.hpp
+            ${PROJECT_BINARY_DIR}/blinkstickcpp_export.h
```
Comment (teekae) [inline blinkstickcpp/CMakeLists.txt]:
```suggestion
            ${PROJECT_BINARY_DIR}/blinkstick/export.hpp
```

File: blinkstickcpp/include/blinkstick/blinkstick.hpp [hunk 3]
```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <blinkstick/device.hpp>
+#include <blinkstickcpp_export.h>
```
Comment (teekae) [inline blinkstickcpp/include/blinkstick/blinkstick.hpp]:
```suggestion
#include <blinkstick/export.hpp>
```

File: blinkstickcpp/include/blinkstick/device.hpp [hunk 4]
```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <blinkstickcpp_export.h>
```
Comment (teekae) [inline blinkstickcpp/include/blinkstick/device.hpp]:
```suggestion
#include <blinkstick/export.hpp>
```

**Review by teekae — CHANGES_REQUESTED** [review]
Apologies for delay in looking at this.

Just suggested a small change to keep the headers consistent

**Comment by ptsouchlos** [comment]
I've made the requested changes. Let me know if there's anything else!

**Review by teekae — APPROVED** [review]

**Pull request merged**